src/new_index/zmq.rs

```diff
@@ -0,0 +1,42 @@
+use bitcoin::{hashes::Hash, BlockHash};
+use crossbeam_channel::Sender;
+
+use crate::util::spawn_thread;
+
+pub fn start(url: &str, block_hash_notify: Option<Sender<BlockHash>>) {
```

Comment:
Does `block_hash_notify` need to be an `Option`?

Reply:
974d78971fbfc9555e7a85fa32b2f91af6db8023